### PR TITLE
French typo

### PR DESCRIPTION
**Pull request by SansPseudoFix**
#1148 continuation (wasn't finish :P).

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -58,21 +58,21 @@ fr:
...
+    prompt_html: 'Vous (<strong>%{self}</strong>) avez demandé à suivre :'
     title: Suivre %{acct}
   datetime:
     distance_in_words:
       about_x_hours: "%{count}h"
-      about_x_months: "%{count}mo"
-      about_x_years: "%{count}y"
-      almost_x_years: "%{count}y"
+      about_x_months: "%{count}mois"
+      about_x_years: "%{count}an(s)"
```
Comment (Gargron) [inline config/locales/fr.yml]: It is possible to localize pluralizations like this:

```yml
about_x_months:
  one: One month
  other: "%{count} months
```

**Review by Gargron — COMMENTED** [review]
I can merge as-is but you can get rid of the ugly "an(s)" by localizing pluralizations (see other comment) if you want, first.

File: config/locales/fr.yml (hunk 1 above)
Comment (SansPseudoFix) [inline config/locales/fr.yml]: Right! I'm on it, thx :)

File: config/locales/fr.yml [hunk 2]
```diff
@@ -58,21 +58,27 @@ fr:
...
   datetime:
     distance_in_words:
       about_x_hours: "%{count}h"
-      about_x_months: "%{count}mo"
-      about_x_years: "%{count}y"
-      almost_x_years: "%{count}y"
+      about_x_months: "%{count}mois"
+      about_x_years:
+        one: un an
+        other: %{count} ans
```
Comment (SansPseudoFix) [inline config/locales/fr.yml]: Like this?

**Comment by nightpool** [comment]
Looks like you have build errors!

```
I18n::InvalidLocaleData:
       can not load translations from /home/travis/build/tootsuite/mastodon/config/locales/fr.yml: 
       Psych::SyntaxError: (/home/travis/build/tootsuite/mastodon/config/locales/fr.yml):
       found character that cannot start any token while scanning for the next
       token at line 69 column 16
```

have you validated your YAML? It looks like you're missing some quotation marks on line 69

**Comment by SansPseudoFix** [comment]
> have you validated your YAML? It looks like you're missing some quotation marks on line 69

 Mh, I didn't valide my code, it would be a good idea indeed…
Now, it's ok: validation, tests, and squash :)

**Review by Gargron — APPROVED** [review]

**Pull request merged**